### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by RaoAkif**
- Initialise React app.
- The building blocks of your app should be set as re-usable components. Create a directory for your components.
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation.
- Styling is not required at this point

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,10 @@
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
     "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-react": "7.28.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-router-dom": "^6.3.0",
```
Comment (Cathella) [inline package.json:13]: ✅ `react-router-dom` should be present in `package.json` 👍🏻.

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <h2 className="logo">Bookstore CMS</h2>
+        <nav>
+          <ul style={navLinks}>
+            <li>
+              <Link className="navLink" to="/">Books</Link>
+            </li>
+            <li>
+              <Link className="navLink" to="/categories">Categories</Link>
+            </li>
+          </ul>
```
Comment (Cathella) [inline src/components/Header/Header.jsx:31]: ✅ Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 👍🏻.

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Header />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
```
Comment (Cathella) [inline src/App.js:18]: ✅ Has 2 `<Route>`s, one for the BookList/Books component and the other for the Categories component 👍🏻.

File: src/pages/Categories.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
```
Comment (Cathella) [inline src/pages/Categories.jsx]: ### 🛑 Required Change
- [x] The Categories component is meant to display a button with text "Check status" 🤷🏻

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @RaoAkif 👋🏻, 

Your CSS skills are top-notch ❤️‍🔥. This shows in the styling of your project which looks amazing 💪🏻! However, there are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Professional README.
✅ The project is initialized using create-react-app
✅ The BookList/Books component is the default page

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Categories.jsx (hunk 4 above)
Comment (RaoAkif) [reply src/pages/Categories.jsx]: Added

**Review by Dipeshtwis — APPROVED** [review]
Hi @RaoAkif, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**